pkg/kritis/admission/admission.go

```diff
@@ -183,7 +183,8 @@ func reviewImages(images []string, ns string, pod *v1.Pod, ar *v1beta1.Admission
 		createDeniedResponse(ar, errMsg)
 		return
 	}
-	r := review.New(client, defaultViolationStrategy, securitypolicy.ValidateImageSecurityPolicy)
+
+	r := review.New(client, defaultViolationStrategy, true, nil)
```

Comment:
If this is going to require more arguments, could it be changed to pass a config struct? It's no longer guessable what these arguments are, and that strategy would allow you to pass very specific hints:

r := review.New(client, &Review{Strategy: default, ILoveFrogs: true, NextDiaperChange: nil})